django/contrib/admin/static/admin/css/forms.css

```diff
@@ -37,26 +37,26 @@ label {
 
 /* RADIO BUTTONS */
 
-form ul.radiolist li {
+form div.radiolist li {
```

Comment:
There will be no `li` element to select since `ul` elements were removed, so this selector is redundant.